wolframalpha/__init__.py

```diff
@@ -77,7 +77,7 @@ def from_doc(cls, doc):
         Load instances from the xmltodict result. Always return
         an iterable, even if the result is a singleton.
         """
-        return map(cls, always_iterable(doc))
+        return map(cls, always_iterable(doc, base_type=dict))
```

Comment:
This change removes the base_types of `str` and `bytes`. I wonder if that's okay in this case.

Reply:
Regardless, I've merged this commit onto a test that replicates the failure. If it turns out that the other singletons are important, we can address that, but I suspect they're not as this is passed to a `dict` constructor.